frontend/src/app/services/load-files/load-files.service.ts

```diff
@@ -29,4 +29,12 @@ export class LoadFilesService {
       this.BACKEND_URL_PREFIX + id + '/files?show_hidden=' + showHiddenFiles
     );
   }
+
+  download(id: string, filename: string): Observable<Blob> {
+    return this.http.request(
+      'GET',
```

Comment:
```suggestion
    return this.http.get(
```

You can just use `http.get` instead.